pkgs/applications/version-management/forgejo/default.nix

```diff
@@ -25,13 +25,11 @@
 
 buildGoModule rec {
   pname = "forgejo";
-  version = "1.19.0-2";
+  version = "1.19.0-3";
 
   src = fetchurl {
-    name = "${pname}-src-${version}.tar.gz";
-    # see https://codeberg.org/forgejo/forgejo/releases
-    url = "https://codeberg.org/attachments/2bf497db-fa91-4260-9c98-5c791b6b397c";
-    hash = "sha256-neDIT+V3qHR8xgP4iy4TJQ6PCWO3svpSA7FLCacQSMI=";
+    url = "https://codeberg.org/forgejo/forgejo/releases/download/v${version}/forgejo-src-${version}.tar.gz";
+    hash = "sha256-u27DDw3JUtVJ2nvkKfaGzpYP8bpRnwc1LUVra8Epkjc=";
   };
```

Comment:
We also have `fetchFromGitea` fwiw.

Reply:
`fetchFromGitea` needs a `rev`, but this URL doesn't. We are using a release artifact/tarball.

I can reach back to explain why, if you are interested.
But the tl;dr; is forgejo provides release tarballs, that already include the frontend.
So we don't need to build everything from source. Similarly to Gitea.

If you go to https://codeberg.org/forgejo/forgejo/releases/tag/v1.19.0-3 you can compare [Source Code (TAR.GZ)](https://codeberg.org/forgejo/forgejo/archive/v1.19.0-3.tar.gz) with [forgejo-src-1.19.0-3.tar.gz](https://codeberg.org/attachments/8538f5e0-fe73-4257-810a-ee6f0339914c)

Reply:
Ideally, we would build the frontend from source. This could be done using `buildNpmPackage`, but should probably happen outside a regular update.